.gitignore

```diff
@@ -74,6 +74,7 @@ missing
 default_commit_hash
 src/s3fs
 src/test_*
+!src/test_string_util.cpp
```

Comment:
Perhaps we should change `src/test_*` to `test_string_util`?

Reply:
I was wondering which one to use, but is that better?
A new force push. soon